BugsnagNetworkRequestPlugin/BugsnagNetworkRequestPlugin/BSGURLSessionTracingDelegate.m

```diff
@@ -72,32 +79,24 @@ - (void)URLSession:(NSURLSession *)session
         NSURLComponents *urlComponents = [NSURLComponents componentsWithURL:req.URL resolvingAgainstBaseURL:YES];
         // Note: Cannot use metrics transaction response because it will be nil if a custom NSURLProtocol is present.
         // Note: If there was an error, task.response will be nil, and the following values will be set accordingly.
-        NSURLResponse *resp = task.response;
-        NSHTTPURLResponse *httpResp = [resp isKindOfClass:NSHTTPURLResponse.class] ? (NSHTTPURLResponse *)resp : nil;
-        NSString *message = [self conclusionForResponseCode:httpResp.statusCode];
-        int64_t requestContentLength = (int64_t)req.HTTPBody.length;
-        int64_t responseContentLength = resp.expectedContentLength;
-
-        if (@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)) {
-            NSURLSessionTaskTransactionMetrics *transaction = [metrics.transactionMetrics lastObject];
-            // Note: Must check for zero because these somtimes lie.
-            if (transaction.countOfRequestBodyBytesSent != 0) {
-                requestContentLength= transaction.countOfRequestBodyBytesSent;
```

Comment:
Is this different from `req.HTTPBody.length`? Seems odd with body in the name

Reply:
The documentation says

```
/*
 * countOfRequestBodyBytesSent is the number of bytes transferred for request body.
 * It includes protocol-specific framing, transfer encoding, and content encoding.
 */
@property (readonly) int64_t countOfRequestBodyBytesSent API_AVAILABLE(macos(10.15), ios(13.0), watchos(6.0), tvos(13.0));
```

so due to including encoding etc it's not the same as the size of the HTTPBody passed in by the app.